examples/app/weather/templates/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>{{ title if title else "Bokeh Plot" }}</title>
+        {{ bokeh_css }}
+        {{ bokeh_js }}
+    </head>
+    <body>
+        <div>
+            HERE IS SOME CUSTOM STUFF
+        </div>
+        {{ plot_div|indent(8) }}
+        {{ plot_script|indent(8) }}
```

Comment:
Yes sorry I should have specified, I am definitely wanting to spread things out over a few independent PRs. This one make a minimal change to add a pretty important capability, do intend to refine it more.